app/views/rescue_codes/new.html.slim

```diff
@@ -0,0 +1,21 @@
+- title t('titles.rescue_codes')
+
+h1.heading = t('headings.rescue_codes')
+p
+  | Below are a set of rescue codes that we have generated for you.
+    You can use these codes to access your account in case you loose
+    access to your phone. Please download these codes and keep them
+    in a safe place.
```

Comment:
Another reason for not providing a download option is that it is only relevant to desktop computers AFAIK, which would mean adding writing some more code to remove the button on mobile devices. I don't think the added complexity is worth it. I would vote for keeping things as simple as possible for now, and only add complexity if absolutely necessary.

Also, at least on a Mac, you can save the codes to your computer, or even to Google Drive, via the Print menu, so if you really wanted to download them, you can by way of printing, which is easy to implement with a handful or less of lines of JavaScript.